### PR TITLE
layers: Group graphics portability validation

### PR DESCRIPTION
**Pull request by spencer-lunarg**
create a `ValidateGraphicsPipelinePortability` to group the `VK_KHR_portability_subset` code

it was split in 2 different areas

### CODE REVIEW

**Comment by spencer-lunarg** [comment]
seems to be failing `VkPositiveLayerTest.DynamicRenderingCreateGraphicsPipeline`on Mac M1 devices  but don't have a way to reproduce it currently

**Comment by juan-lunarg** [comment]
> seems to be failing `VkPositiveLayerTest.DynamicRenderingCreateGraphicsPipeline`on Mac M1 devices but don't have a way to reproduce it currently

Perhaps setting `VK_KHRONOS_PROFILES_EMULATE_PORTABILITY` to `true`?

**Comment by spencer-lunarg** [comment]
> Perhaps setting VK_KHRONOS_PROFILES_EMULATE_PORTABILITY to true?

thought and tried, didn't work, might be a good excuse to get portability working for Max Profile :smile:

File: layers/core_checks/pipeline_validation.cpp [hunk 1]
```diff
@@ -212,6 +161,13 @@ bool CoreChecks::ValidatePipeline(const PIPELINE_STATE &pipeline) const {
                              pipeline.create_index, subpass, rp_state->createInfo.subpassCount - 1);
             subpass_desc = nullptr;
         }
+
+        // Check for portability errors
+        // Issue raised in https://gitlab.khronos.org/vulkan/vulkan/-/issues/3436
+        // The combination of GPL/DynaimcRendering and Portabilty has spec issues needing to be clairifed
```
Comment (juan-lunarg) [inline layers/core_checks/pipeline_validation.cpp:167]:
```suggestion
        // The combination of GPL/DynamicRendering and Portability has spec issues that need to be clarified
```

**Review by juan-lunarg — APPROVED** [review]

**Review by arno-lunarg — APPROVED** [review]

**Review by ncesario-lunarg — APPROVED** [review]

**Pull request merged**